modules/cluster-logging-collector-log-forward-examples.adoc

```diff
@@ -0,0 +1,385 @@
+** `10` or `authpriv` for security/authentication messages.
+** `11` or `ftp` for the FTP daemon.
+** `12` or `ntp` for the NTP subsystem.
+** `13` or `security` for log audit.
+** `14` or `console` for log alert.
+** `15` or `solaris-cron` for the scheduling daemon.
+** `16`–`23` or `local0` – `local7` for locally used facilities
+* payloadKey: The record field to use as payload for the syslog message.
+* rfc: The RFC to be used for sending log using syslog. The default is RFC5424.
+* severity: The link:https://tools.ietf.org/html/rfc5424#section-6.2.1[syslog severity] to set on outgoing syslog records. The value can be a decimal integer or a case-insensitive keyword:
+** `0` or `Emergency` for messages indicating the system is unusable.
+** `1` or `Alert` for messages indicating action must be taken immediately.
+** `2` or `Critical` for messages indicating critical conditions.
+** `3` or `Error` for messages indicating error conditions.
+** `4` or `Warning` for messages indicating warning conditions.
+** `5` or `Notice` for messages indicating normal but significant condition.
+** `6` or `Informational` for messages indicating informational messages.
+** `7` or `Debug` for messages indicating debug-level messages. This is the default.
+* tag: Tag specifies a record field to use as tag on the syslog message.
+* trimPrefix: Remove the specified prefix from the tag.
+
+[id=cluster-logging-collector-log-forward-examples-syslog-5424]
+=== Additional RFC5424 syslog parameters
+
+The following parameters apply to RFC5424:
+
+* appName: The APP-NAME part of the syslog-msg header. Must be specified for `RFC5424`.
+* msgID: The MSGID part of the syslog-msg header. Must be specified for `RFC5424`.
+* procID: The PROCID part of the syslog-msg header. Must be specified for `RFC5424`.
+
+[id=cluster-logging-collector-log-forward-examples-secure]
+== Forwarding logs using secure mechanisms
+
+The following example forwards logs using multiple security methods:
+
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "LogForwarding"
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+   - name: elasticsearch-secure
+     type: "elasticsearch"
+     url: https://elasticsearch.sample.example.com:9200 <1>
+   - name: elasticsearch-secret
+     type: "elasticsearch"
+     url: https://elasticsearch.secret.example.com:9200
+     secret:
+        name: es-secret <2>
+   - name: elasticsearch-insecure
+     type: "elasticsearch"
+     url: elasticsearch.insecure.example.com.svc:9200
+  pipelines:
+   - name: infrastructure-logs
+     inputRefs:
+     - infrastructure
+     outputRefs:
+     - elasticsearch-secure
+   - name: application-logs
+     inputRefs:
+     - application
+     outputRefs:
+     - elasticsearch-secret
+   - name: audit-logs
+     inputRefs:
+     - audit
+     outputRefs:
+     - elasticsearch-insecure
+----
+<1> To use server-authenticated TLS, specify a secure URL, with the `https` prefix.
```

Comment:
Per my understanding, no matter what kind of authentication the forward use, if user want to enable TLS, then they must create a secret and specify it in the cluster log forwarder CR. @alanconway could you please help to confirm this?

Reply:
Removed this topic.